### ISSUE TITLE
Any plan for upgrade to bootstrap 4?

### ISSUE THREAD

**Issue by su-xiaolin** [timcreative/get-shit-done#3]

**Comment by timshingyu** [comment]
Yes, we will upgrade it once bootstrap 4 is officially released. Also, it will be the free upgrade as well. So if you purchase it now, you can enjoy the free upgrade to bootstrap 4.
